content/runbooks/rds/RDSCACertificateCloseToExpiration.md

```diff
@@ -0,0 +1,59 @@
+---
+title: CA Certificate Close to Expiration
+---
+
+# RDSCACertificateCloseToExpiration
+
+## Meaning
+
+Alert is triggered when an RDS instance is detected using a CA certificate which is going to expire in less than 15 days.
+
+## Impact
+
+If the certificate is not renewed before expiration, all attempts to initiate an SSL/TLS connection to the RDS instance will fail.
+
```

Comment:
Majority of the alerts will be related to August 2024 certificate expiration, I would suggest a specific disclamer

```suggestion

{{< hint warning >}}
**Important**

The `Amazon RDS Root 2019 CA` certificate expires on **Aug 22 17:08:50 2024 GMT**.

- Starting January 25th 2024, RDS instances created without specifying the CA will use `rds-ca-rsa2048-g1``.
- In August 2024, AWS will enforce the CA rotation on all RDS instances on the expiring CA during a window maintenance
{{< /hint >}}

```

Reply:
```suggestion
**Aug 22 17:08:50 2024 UTC**
```
Small nitpick but I'd recommend using UTC over any local timezone for better internationalisation.

Reply:
done 👍